csrc/adam/fused_adam_frontend.cpp

```diff
@@ -1,5 +1,21 @@
 #include <torch/extension.h>
 
+#ifdef SYCL_KERNEL
```

Comment:
Needs to remove these SYCL specific code

Reply:
There codes will be removed when Intel Extension for DeepSpeed replicate this file